source/common/http/http2/codec_impl.h

```diff
@@ -258,8 +261,8 @@ class ConnectionImpl : public virtual Connection,
     }
 
     void setWriteBufferWatermarks(uint32_t low_watermark, uint32_t high_watermark) {
-      pending_recv_data_.setWatermarks(low_watermark, high_watermark);
-      pending_send_data_.setWatermarks(low_watermark, high_watermark);
+      pending_recv_data_->setWatermarks(low_watermark, high_watermark);
+      pending_send_data_->setWatermarks(low_watermark, high_watermark);
```

Comment:
This should really transition to the 1 argument version of setWatermarks.  There's only 1 call to setWriteBufferWatermarks and it uses `buffer_limit / 2, buffer_limit` as arguments.